daemon/sdnotify.go

```diff
@@ -2,30 +2,33 @@
 package daemon
 
 import (
-	"errors"
 	"net"
 	"os"
 )
 
-var SdNotifyNoSocket = errors.New("No socket")
-
 // SdNotify sends a message to the init daemon. It is common to ignore the error.
```

Comment:
I just read @alban's concern about the clumsiness of my proposal. Well, it may seem clumsy, but that's quite "idiomatic" in Go, the most prominent example being the const `io.EOF`, amongst other possible solutions (see http://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully).

To make it more prominent, we could/should make the behavior very explicit in the godoc.